tests/integration/test_alias.py

```diff
@@ -0,0 +1,93 @@
+import csv
+import shutil
+from traceback import print_exception
+
+import pytest
+from click.testing import CliRunner
+
+from cromshell.__main__ import main_entry as cromshell
+
+
+def run_cromshell_alias(workflow_id: str, alias_name: str, exit_code: int):
+    """Run cromshell alias using CliRunner and assert job is successful"""
+
+    runner = CliRunner(mix_stderr=False)
+    # The absolute path will be passed to the invoke command because
+    # the test is being run in temp directory created by CliRunner.
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            cromshell,
+            [
+                "alias",
+                workflow_id,
+                alias_name,
+            ],
+        )
+        assert result.exit_code == exit_code, (
+            f"\nSTDOUT:\n{result.stdout}"
+            f"\nSTDERR:\n{result.stderr}"
+            f"\nExceptions:\n{result.exception}"
+            f"\n{print_exception(*result.exc_info)}"
+        )
+        return result
+
+
+def assert_workflow_id_matches_alias(
+    workflow_id: str, alias_name: str, local_workflow_database_tsv: str
+):
+    """
+    For a given workflow id assert that the alias matches a given alias name
+    """
+    with open(local_workflow_database_tsv, "r") as csv_file:
+        reader = csv.DictReader(csv_file, delimiter="\t")
+        for row in reader:
+            if row["RUN_ID"] == workflow_id:
+                #  If alias_name is "" (which is when we want to remove an alias) assert
+                #  that the alias for the workflow id returns None. Else (when adding
+                #  and alias) assert that the alias for the workflow id returns
+                #  the matching alias name.
+                if alias_name == "":
+                    assert row["ALIAS"] is None
+                else:
+                    assert row["ALIAS"] == alias_name
+
+
+class TestAlias:
+    @pytest.mark.parametrize(
+        "workflow_id, alias_name, exit_code",
+        [
```

Comment:
given that the `alias` commands take -1, -2 values, this test should be expanded to cover that.

Reply:
done